packages/smonit/files/etc/smonit/dnsmasq

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+hook_name="dnsmasq"
+
+function hook_is_installed() {
+  installed=no
+  [ -x /usr/sbin/dnsmasq ] && [ -x /etc/init.d/dnsmasq ] && installed=yes                                         
+}   
+
+function hook_check() {
```

Comment:
It is not enough. It is just an "initial" hook (almost an exampe), to help understand the concept.